content/blog/2023-06-01-ethereum-classic-course-24-what-are-programmable-native-cryptocurrencies/index.md

```diff
@@ -0,0 +1,76 @@
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/WvGttc5lAGI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+In the previous class of the Ethereum Classic (ETC) course we explained what are the native cryptocurrencies, the original invention of the industry, how they gain economic value, and why the proof of work versions of them are digital gold.
+
+In this class we will explain what are programmable digital cryptocurrencies, why ETC is called programmable digital gold, and why it has an enhanced level of trust minimization as compared to non-programmable digital gold.
+
+In the next few classes we will cover these other topics:
+
+- Class 25: What Are ERC-20 Tokens?
+- Class 26: What Are Convertible Stablecoins?
+- Class 27: What Are Algorithmic Stablecoins?
+- Class 28: What Are Privacy Coins?
+- Class 29: What Are Decentralized Finance (DeFi) Coins?
+- Class 30: What Are File Storage Coins?
+
+## What Are Non-Programmable Native Cryptocurrencies?
+
+As we explained in the previous class, native cryptocurrencies are the coins that are issued to pay for miner rewards and for fees to have transactions included in blocks. They are designed into the base protocol of a blockchain network and have economic value because of their default use cases.
+
+Of all the native cryptocurrencies some are non-programmable and others are programmable. 
+
+Bitcoin, for example, is merely a ledger with accounts and balances and people can only move coins from one account to the other in that system. This is a model of a non-programmable coin because inside Bitcoin it is impossible to host decentralized software programs.
+
+Other non-programmable coins are Litecoin, Monero, and DogeCoin. 
+
+## What Are Programmable Native Cryptocurrencies?
+
+On the other hand, Ethereum Classic is an example of a programmable coin. ETC is the same as Bitcoin in that it is a ledger with accounts and balances, but it also has space in its ledger to store decentralized software programs.
+
+These decentralized software programs can have their own ETC addresses, hold ETC, move ETC when called, and do all these things with complex rules and conditions. This is what makes the ETC coin a programmable native cryptocurrency.
+
+Inside ETC it is possible to have smart contracts with all sorts of programmability and functions, from banking, to insurance, to decentralized exchanges, property registries, NFTs, decentralized auctions, bonds and stock issuance, DAOs, and many more kinds of use cases.
+
+## What Is Programmable Digital Gold?
+
+The quality of digital gold is something that both Bitcoin and ETC enjoy because both coins are issued only after enormous amounts of computational work is done by miners, thus making them scarce and valuable, just like gold is in the real world.
```

Comment:
Linde 50: Verb agreement; Suggested wording change
```suggestion
The quality of digital gold is something that both Bitcoin and ETC enjoy because both coins are issued only after enormous amounts of computational work are done by miners, thus making them scarce and valuable, just like gold is in the physical world.
```